lib/mayaUsd/ufe/UsdUndoRenameCommand.cpp

```diff
@@ -122,7 +125,14 @@ bool UsdUndoRenameCommand::renameRedo()
     // the renamed scene item is a "sibling" of its original name.
     _ufeDstItem = createSiblingSceneItem(_ufeSrcItem->path(), _newName);
     sendRenameNotification(_ufeDstItem, _ufeSrcItem->path());
- 
+
+    // SdfLayer is a "simple" container, and all it knows about defaultPrim is that it is a piece of token-valued layer metadata.  
+    // It is only the higher-level Usd and Pcp modules that know that it is identifying a prim on the stage.  
+    // One must use the SdfLayer API for setting the defaultPrim when you rename the prim it identifies.
+    if(primNameStr == stageDefPrimNameStr){
```

Comment:
@ppt-adsk We don't need to do a comparison here.  if(stageDefPrimNameStr) does the work.  Will fix it in the next commit,

Reply:
Hmm, I don't think that would work now that I think about it. please ignore my last comment.

Reply:
@ppt-adsk please see d84d82fcaa2f377e77114aaf9adf25f5be8c601b